modules/installation-gcp-user-infra-adding-ingress.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-gcp-user-infra-adding-ingress_{context}"]
+= Optional: Adding the ingress DNS records
+
+If you removed the DNS Zone configuration when creating Kubernetes manifests and
+generating Ignition configs, you must manually create DNS records pointing at
+the ingress load balancer. You can create either a wildcard
+`*.apps.{baseDomain}.` or specific records. You can use A, CNAME, and other
+records per your requirements.
+
+.Prerequisites
+
+* Configure a GCP account.
+* Remove the DNS Zone configuration when creating Kubernetes manifests and
+generating Ignition configs.
+* Create and configure a VPC and assocated subnets in GCP.
+* Create and configure networking and load balancers in GCP.
+* Create control plane and compute roles.
+* Create the bootstrap machine.
+* Create the control plane machines.
+* Create the worker machines.
+
+.Procedure
+
+. Wait for the Ingress router to create a load balancer and populate the `EXTERNAL-IP` field:
++
+----
+$ oc -n openshift-ingress get service router-default
+NAME             TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                      AGE
+router-default   LoadBalancer   172.30.18.154   35.233.157.184   80:32288/TCP,443:31215/TCP   98
+----
+
+. Create the A record to your public and private zones:
```

Comment:
nit: `Create the A record to your public and private zones` -> `Create the A record in your public and private zones` or `Add the A record to your public and private zones`

Reply:
I'm suggesting "Add" upstream: openshift/installer#2466